### DISCUSSION TITLE
Changelog retrieval behind corporate proxy fails with timeout

### DISCUSSION THREAD

**Discussion by wolf-kaefer** [renovatebot/renovate#18877] [Request Help] [answered]
We are setting up a renovate bot (self hosted, using the npm package) behind a corporate proxy. Seems to work fine so far, only the changelog retrieval from github fails with a timout. I'm assuming the corporate proxy blocks/interferes with the request (unverified, I just naturally blame the proxy ...). 

If so, which proxy settings does renovate use there?  I would hope/expect the ones from my .npmrc, especially since i have configured `detectGlobalManagerConfig: true`? Those usually work though...

Log snippet  - I can provide more, if necessary:
"message": "connect ETIMEDOUT 140.82.121.6:443"
 "url": "https://api.github.com/repos/junit-team/junit5/tags?per_page=100",
 
 (Side node: A github token is configured, and i would assume an authentication error of some kind if it is not valid or missing)

**Comment by rarkins** [comment, marked as answer]
Renovate understands HTTP_PROXY, HTTPS_PROXY, NO_PROXY environment variables

**Reply by wolf-kaefer** [reply]
thanks - that did the trick.
